bdwgraph.cpp

```diff
@@ -135,3 +135,77 @@ void BDWGraph::drawZeroAxis()
     
     qDebug() << "BDW zero axis drawn at x:" << zeroPoint.x();
 }
+
+/**
+ * @brief Override drawDataLine to use dashed lines for BDW graph
+ *
+ */
+void BDWGraph::drawDataLine(const QString &seriesLabel, bool plotPoints)
```

Comment:
To me it looks like this needs to be included into the waterfall graph with the corresponding method arguments.

I'm curious to know what the difference is with the rest of the codebase.